src/test/java/com/microsoft/sqlserver/jdbc/datatypes/SQLServerSpatialDatatypeTest.java

```diff
@@ -821,7 +1842,7 @@ public void testParse() throws SQLException {
 
     @Test
     @Tag(Constants.xAzureSQLDW)
-    public void testPoint() throws SQLException {
+    public void testSendBasicPoint() throws SQLException {
         beforeEachSetup();
```

Comment:
can we rename this, as initially I thought this was a hack to avoid @BeforeEach. But actually this isn't called before each tests.

Reply:
and, so if these table names are only used in the before each test which use them, then it shouldn't be in the @BeforeAll method right? (lines 2273-2275) Should move to this beforeEachSetup() method instead. And, can we do the escape quotes at the same time we're generating the names so we don't have to do it everywhere?